openwisp_controller/config/migrations/0061_remove_whoisinfo_country_and_more.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 5.2.1 on 2025-06-17 16:51
```

Comment:
Let's have a single migration file only. Add changes done in this file in first file only

Reply:
@devkapilbansal we have deployed this in staging, we need to provide incremental migrations. We can squash the migrations before merging.